components/src/preact/mutationsOverTime/mutations-over-time.tsx

```diff
@@ -154,13 +155,16 @@ const MutationsOverTimeTabs: FunctionComponent<MutationOverTimeTabsProps> = ({
         { label: 'Deletions', checked: true, type: 'deletion' },
     ]);
 
+    const [gapsHidden, setGapsHidden] = useState<boolean>(originalComponentProps.initialGapsHidden ?? false);
```

Comment:
Nitpick: I find this name a little more intuititve.
```suggestion
    const [hideGaps, setHideGaps] = useState<boolean>(originalComponentProps.initialGapsHidden ?? false);
```